tests/api/test_api_v2.py

```diff
@@ -54,16 +56,16 @@ def test_error_exception(self, accept_json):
 
     def test_error_status_401(self, accept_json, monkeypatch):
         """Test the /error endpoint. Function Redirect."""
-        monkeypatch.setenv("REDIRECT_STATUS", '401')
+        monkeypatch.setenv('REDIRECT_STATUS', '401')
         response = self.client.get(api_route_for('/_error'), headers=accept_json)
         assert response.status_code == 401
-        monkeypatch.delenv("REDIRECT_STATUS", raising=False)
+        monkeypatch.delenv('REDIRECT_STATUS', raising=False)
```

Comment:
delenv is not required. pytest makes sure to undo during the test tear-down, which happens after completion of each test.

Reply:
This is fixed in commit 'Review comment to HTTPError from base classes is fixed.'